ui/perfherder/alerts/AlertActionPanel.jsx

```diff
@@ -169,23 +164,7 @@ export default class AlertActionPanel extends React.Component {
             </Col>
           )}
 
-          {!this.hasTriagedAlerts() && !this.allAlertsConfirming() && (
-            <Col sm="auto" className="p-2">
-              <SimpleTooltip
-                text={
-                  <Button
-                    color="secondary"
-                    onClick={() => this.updateAlerts('confirming')}
-                  >
-                    <FontAwesomeIcon icon={faClock} /> Confirming
```

Comment:
`faClock` import from this file should be removed
CI job fails because of this import: https://travis-ci.org/mozilla/treeherder/jobs/587443376

```error: 'faClock' is defined but never used (no-unused-vars) at ui/perfherder/alerts/AlertActionPanel.jsx:6:3:
  4 | import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
  5 | import {
 6 |   faClock,
    |   ^
  7 |   faCheck,
  8 |   faBan,
  9 |   faLevelDownAlt,```